### ISSUE TITLE
Always show temperature and keep display on in idle mode if the tip is still hot.

### ISSUE THREAD

**Issue by JohnEdwa** [Ralim/ts100#187]
Right now if I shut the iron off and cancel out of the cooldown screen, it'll go back to the idle screen that doesn't show the tip temperature. Even if you have 'Detailed Idle Screen' toggled, it'll shortly shut the screen off even when the tip is still dangerously hot (190C in my case).

Id's suggest always disabling screen sleep if the tip is over 50C and implementing the tip temperature warning to the default idle screen as well.   
Maybe replace the "soldering mode" icon with the exclamation triangle or simply with the tip display?

**Comment by Ralim** [comment]
This should be fixed in the form that it wont let the screen goto sleep etc.
I haven't added the tip display yet. Not sure the best easy way to do this without encoding extra graphics yet.

**Comment by JohnEdwa** [comment]
I'd just write the temperature where the 'start soldering' icon usually is until it got cool enough to be save (so... under 50C I guess). 
Something like this:
![menu_temp](https://user-images.githubusercontent.com/5417183/37248731-e5634b12-24e1-11e8-9ecb-020b81e3123a.png)

Btw, do you have the icons, fonts and other resources available somewhere so I can start making better-looking mockups :P ?

**Comment by Ralim** [comment]
Yeah thats basically what im looking at doing now.

Not really, some of the icons are available in the issues that created them, for the ones I created i just draw them using a binary-> hex converter and time :)

**Comment by JohnEdwa** [comment]
Oof, no kidding. When you try to convert them back to [XBitmaps](https://en.wikipedia.org/wiki/X_BitMap), this is your 'WarningBlock24': ![output](https://user-images.githubusercontent.com/5417183/37249605-613e0c20-24f3-11e8-9a31-9f8dc3425e97.png) Instead of left-to-right, you encode them vertically, split into two, that are mirrored and in the wrong order :P

Oh well, I got to hone my almost none-existant python skills again and managed to construct a script that converts them back successfully, so here they are:
![extrafontchars_1](https://user-images.githubusercontent.com/5417183/37250296-048ddb68-2502-11e8-9455-b7f40ad47092.png)
![extrafontchars_2](https://user-images.githubusercontent.com/5417183/37250297-04b0158e-2502-11e8-972d-243b0eb392cc.png)
![font_6x8_cyrillic](https://user-images.githubusercontent.com/5417183/37250298-04d1415a-2502-11e8-886e-32c969c20d22.png)
![font_6x8_latin_1](https://user-images.githubusercontent.com/5417183/37250299-04f2d068-2502-11e8-991b-8a4f840b4706.png)
![font_6x8_latin_ext](https://user-images.githubusercontent.com/5417183/37250300-051d2d2c-2502-11e8-87e1-e95ff01fd0e3.png)
![font12_cyrillic](https://user-images.githubusercontent.com/5417183/37250301-056ec560-2502-11e8-9ac8-7fccd5b0d730.png)
![font12_latin_1](https://user-images.githubusercontent.com/5417183/37250302-05d16738-2502-11e8-93c3-1e615722e9ef.png)
![font12_latin_2](https://user-images.githubusercontent.com/5417183/37250303-060d3c68-2502-11e8-9008-52d06e62893b.png)
![font12_latin_ext](https://user-images.githubusercontent.com/5417183/37250304-0654c22c-2502-11e8-8ead-5fcaedf0922b.png)
![fontsymbols](https://user-images.githubusercontent.com/5417183/37250305-0675d99e-2502-11e8-80aa-5ebdbcf30ad3.png)
![idlescreenbg](https://user-images.githubusercontent.com/5417183/37250306-06977414-2502-11e8-8896-f5aee8e540ce.png)
![idlescreenbgf](https://user-images.githubusercontent.com/5417183/37250307-06bbe3c6-2502-11e8-9b4e-daff9895677e.png)
![warningblock24](https://user-images.githubusercontent.com/5417183/37250308-06dcae26-2502-11e8-8e72-f2c77dfdd987.png)

If you could stick these into the 'Development Resources' folder, and save any new icons you come up with, I'd appreciate it.

**Comment by Ralim** [comment]
Yeah they are in the native format for the screen, which is odd if your not used to B&W screens.

Thank you :)

Do you want to add them using a pull request? If not I'll add them :)

Thanks for converting them back :)

Also there should now be a public branch that implements this (drawing the temp), I can compile a test firmware if you would like.

**Comment by JohnEdwa** [comment]
You can add them, it's faster that way :)

And a test build would be neat, but you really should release a new alpha ASAP to fix #223 especially as the old stable isn't compatible with the new accelerometer.

**Comment by Ralim** [comment]
@JohnEdwa I'm gong to try and fix up the #214 concerns and then push one out. 😄 
Doesnt help that it seems most commenters are on the other half of the world, so I wake up to notifications :)

**Comment by Ralim** [comment]
@JohnEdwa Pushed a new release tester out, if no-one has any big issues ill make it a full release and then go from there.
